pkgs/by-name/li/lib60870/package.nix

```diff
@@ -0,0 +1,30 @@
+{ cmake
+, lib
+, stdenv
+, fetchFromGitHub
+}:
+stdenv.mkDerivation {
+  pname = "lib60870";
+  version = "2.3.2";
+
+  src = fetchFromGitHub {
+    owner = "mz-automation";
+    repo = "lib60870";
+    rev = "53a6b3c1cf3023e51cf81763b1ccf048edcd1c64";
+    hash = "sha256-9o+gWQbpCJb+UZzPNmzGqpWD0QbGjg41is/f1POUEQs=";
+  };
+
+  separateDebugInfo = true;
+
+  nativeBuildInputs = [ cmake ];
+
+  preConfigure = "cd lib60870-C";
+
+  meta = with lib; {
+    description = "Implementation of the IEC 60870-5-101/104 protocol";
+    homepage = "https://libiec61850.com/";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ stv0g ];
+    platforms = [ "x86_64-linux" ];
+  };
+}
```

Comment:
```suggestion
})
```

Reply:
Thanks. Addressed in 1a4591f71467ef88dedd2a7f7ba246abd185f1dd